data/zhongwen/articles/c3xd4x9prgyo/trad.json

```diff
@@ -154,6 +154,18 @@
       },
       "metadata": {
         "id": "urn:bbc:ares::article:c3xd4x9prgyo",
+        "atiAnalytics": {
+          "categoryName": null,
```

Comment:
More of a question of the BFF API, is there a reason we're passing these down as null, when we could just omit the key/value pair from the data?

Reply:
The `null` values in the BFF are for scenarios where a specified property in the
`fetchArticle` response is absent. `null` is meant to be the default value for
missing properties instead of setting them to `undefined`.
If understand your query correctly, are you recommending addition of a step in the BFF
payload transformation logic, to remove remove null key/pair values?

Reply:
Potentially, I guess it depends if `null` is needed on the front end, also in metadata I don't think I've seen null values before. If not, is there a downside to removing them? Like I said only a small suggestion, doesn't stop this PR